api/converter/converter_test.go

```diff
@@ -67,6 +67,7 @@ func TestConverter(t *testing.T) {
 		err := doc.Update(func(root *proxy.ObjectProxy) error {
 			// an object and primitive types
 			root.SetNewObject("k1").
+				SetNull("k.1.0").
```

Comment:
I'm curious about "k.1.0".  🤔 Is it different with "k1.0"?

Reply:
@mojosoeun No. It's just a key. I will change it to the same pattern as the other keys.